.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+      interval: "weekly"
```

```suggestion
  schedule:
      interval: "weekly"
```
Indentation fix, and it's fine to check daily 🙂:
```suggestion
  schedule:
      interval: "daily"
```